kyaml/setters2/settersutil/fieldsetter.go

```diff
@@ -47,6 +50,17 @@ func (fs FieldSetter) Set(openAPIPath, resourcesPath string) (int, error) {
 		Description: fs.Description,
 		SetBy:       fs.SetBy,
 	}
+
+	stat, err := os.Stat(openAPIPath)
```

Comment:
Can we add some comments here to explain why we are doing this?

Reply:
Done.